networking.md

```diff
@@ -10,54 +10,51 @@ sockets, see [dart:io][dartio].
 * TOC Placeholder
 {:toc}
 
-## Using the `http` package {#httpdependency}
-
-Flutter supports the [http package][http], version `0.11.3+12` or later.
-
-Start by declaring a dependency on `http` in `pubspec.yaml`:
-
-```
-dependencies:
-  ...
-  http: '>=0.11.3+12'
-```
-
 ## Making HTTP requests
 
-Next, create an HTTP [Client][client]. We recommend using
-[`createHttpClient`](https://docs.flutter.io/flutter/services/createHttpClient.html)
```

Comment:
should we add a section to this doc for mocking/testing with HTTP?

Reply:
We should, and I started doing it yesterday, but looks like the overrides APIs are still in review, so I'd rather do it in a second PR. Filed https://github.com/flutter/flutter/issues/13792 for that.